pandas/_libs/tslibs/timestamps.pyi

```diff
@@ -59,10 +59,6 @@ class Timestamp(datetime):
     # While Timestamp can return pd.NaT, having the constructor return
     # a Union with NaTType makes things awkward for users of pandas
     def _set_freq(self, freq: BaseOffset | None) -> None: ...
-    @classmethod
```

Comment:
```
>>> import pandas
>>> pandas._libs.tslibs.timestamps._Timestamp._from_value_and_reso
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'pandas._libs.tslibs.timestamps._Timestamp' has no attribute '_from_value_and_reso'
```
I do see code for this on the _Timestamp base class in the Cython file. But I'm familiar enough with Cython to know what's going on here — either way seems like it shouldn't be in the pyi file.

Reply:
Okay, I was just comparing to released version of pandas, this was added in #47299